contracts/Ownable.sol

```diff
@@ -0,0 +1,14 @@
+pragma solidity >=0.4.21 <0.6.0;
+
+contract Ownable {
+  address private owner;
```

Comment:
How is a non-expert user supposed to know who the owner is?

Reply:
thanks, please see d3c8992df64b513707d26466349ab257c5cfd8df